src/Aspire.Hosting.Milvus/MilvusContainerImageTags.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Milvus;
+
+internal static class MilvusContainerImageTags
+{
+    public const string Registry = "docker.io";
+    public const string Image = "milvusdb/milvus";
+    public const string Tag = "v2.3.15";
+
+    public const string AttuImage = "zilliz/attu";
+    public const string AttuTag = "v2.3.10";
```

Comment:
Can we ask the owner of this container to start making `vX.Y` versioned tags?

Reply:
https://github.com/zilliztech/attu/issues/502